src/AspNetCore.Identity.MongoDB/MongoUserStore.cs

```diff
@@ -71,7 +71,7 @@ public MongoUserStore(IMongoDatabase database, ILoggerFactory loggerFactory, str
 
             cancellationToken.ThrowIfCancellationRequested();
 
-            await _usersCollection.InsertOneAsync(user, cancellationToken).ConfigureAwait(false);
+            await _usersCollection.InsertOneAsync(user, null, cancellationToken).ConfigureAwait(false);
```

Comment:
that makes sense. However, I will change the usage to below one which seems a bit cleaner:

```
await _usersCollection.InsertOneAsync(user, cancellationToken: cancellationToken).ConfigureAwait(false);
```